### ISSUE TITLE
Remember me

### ISSUE THREAD

**Issue by FlxMgdnz** [teamhanko/hanko#1656]
### Checklist

- [X] I could not find a solution in the existing issues or docs.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/teamhanko/hanko/blob/main/CODE_OF_CONDUCT.md).

### Description

Users should have the option of ending their session automatically when the browser is closed. Depending on the config, in the current implementation either the frontend SDK pulls the JWT from the x-auth-token header or the backend stores the JWT as cookie. In both cases, the session will be active as long as the JWT is valid.

### Describe your ideal solution

As a new, optional feature, we can introduce a Remember me checkbox on the login page in Hanko Elements. When enabled, the user can control via the checkbox whether the session should remain valid for the duration of the JWT (old behavior) or whether the JWT should be stored as a session cookie and deleted when the browser is closed.

Proposed new backend config option:
```
session:
  enable_remember_me: true / false (Default: false)
```

### Hanko Version

1.0.2

**Comment by Mujhtech** [comment]
Hey @FlxMgdnz, I would love to work on this issue. Can you assign it to me?

**Comment by FlxMgdnz** [comment]
Hey @Mujhtech I've added a bit more information to the issue description.

**Comment by Mujhtech** [comment]
<img width="200" alt="Screenshot 2024-10-07 at 13 36 56" src="https://github.com/user-attachments/assets/d357e5ca-5291-46a8-888a-394f969fb840">

@FlxMgdnz I have made few progress by adding enable_remember_me to session config

```go
type Session struct {
	// ...other session config data
	// `enable_remember_me` determines whether remember me functionality should be enable in hanko element ui
	// which allows users to control the session duration whether the JWT should be store in session cookie and deleted
	// when browser is closed or not.
	EnableRememberMe bool `yaml:"enable_remember_me" json:"enable_remember_me,omitempty" koanf:"enable_remember_me" split_words:"true" jsonschema:"default=false"`
}
```

When the value is true, it add remember action to the login init state else suspend remember me action.

What do you think?

**Comment by Mujhtech** [comment]
Also, this is what I noticed: the current implementation is using js-cookie to store the token. By my testing, I experienced that if the tab is open while I close the browser, the cookie stays but if I close the tab before closing the browser, the cookie expires

**Comment by FlxMgdnz** [comment]
Implemented in #1982
